src/com/yetanalytics/lrs_admin_ui/handlers.cljs

```diff
@@ -101,7 +101,9 @@
                        ::db/enable-reactions enable-reactions
                        ::db/enable-admin-delete-actor enable-admin-delete-actor
                        ::db/pref-lang (keyword admin-language-code)
-                       ::db/language (merge lang/language custom-language))
+                       ::db/language (merge-with merge 
+                                                 lang/language 
+                                                 custom-language))
```

Comment:
This will just merge maps one level deeper - it's not _infinitely recursive_ (which I don't think we want anyways), right?

Reply:
yeah, langmaps only go the one.